react/testFromStyleguidist.js

```diff
@@ -134,7 +135,8 @@ const testFromStyleguidist = (name, markdown, require) => {
       Tabs,
       Text,
       Textarea,
-      Toggle
+      Toggle,
+      true
```

Comment:
I may miss something out, but why is it just `true` here?

Reply:
This is the value of the isTesting option. Here the function that will execute the code inside the codeblocks is defined. For example if you have a code block in the Readme.md like that : 

```
<Button>My Button</Button>
```

, it gets wrapped inside this function like that : 

```
function (require, ..., Button, ... Toggle, isTesting) {
  <Button>MyButton</Button>
}
```

and this function is bound so that each argument is correctly defined. Here the `true` is the value bound to the argument `isTesting` so that you can use `isTesting` inside your codeblocks, in the jest context. In the styleguide context, this is achieved via the context option of styleguidist.

The `isTesting` variable is useful to have the selectBox opened when snapshotting, so that `<options>` are snapshotted.